layouts/basic.vue

```diff
@@ -26,12 +26,24 @@ body {
 @tailwind base;
 
 h1 {
-  @apply text-2xl;
+  font-family: "EB Garamond", sans-serif;
```

Comment:
Sorry Ruby I said this was done but I've just realised we are using Tailwind. We should be setting these header scales in the tailwind config so we can then use text-2xl etc as above.

To be honest, I'm quite surprised these aren't set already as Tailwind has pretty good header size defaults. I guess we have either removed them, not set up Tailwind properly, or are overriding them. Could you take a look into this please? Thank you!

Reply:
have changed to use tailwind

Reply:
Thank you.